### PR TITLE
windows: Various small documentation fixes

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
Various small documentation fixes.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

win_command, win_nssm, win_region, win_shell, win_updates, win_webipcmd

##### ANSIBLE VERSION

v2.3

### CODE REVIEW

File: lib/ansible/modules/windows/win_webpicmd.py [hunk 1]
```diff
@@ -33,11 +33,11 @@
 short_description: Installs packages using Web Platform Installer command-line
 description:
     - Installs packages using Web Platform Installer command-line
-      (http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release).
-    - Must be installed and present in PATH (see win_chocolatey module; 'webpicmd' is the package name, and you must install 'lessmsi' first too)
-    - Install IIS first (see win_feature module)
+      (U(http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release)).
```
Comment (jhawkesworth) [inline lib/ansible/modules/windows/win_webpicmd.py:36]: This is still a bit confusing.  How about putting the things that need doing first first in the list?  I have not used so I'm guessing here, but how about something like this?

Before using this module, ensure that you have
Installed IIS ( see M(win_feature))
Installed 'lessmsi' and 'webpicmd'.  Both 'lessmsi' and 'webpicmd' can be installed using M(win_chocolatey).
Alternatively (U(http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release)).

**Review by dharmabumstead — APPROVED** [review]
Docs are OK. @nitzmahone, please have a look.

File: lib/ansible/modules/windows/win_webpicmd.py (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/windows/win_webpicmd.py:36]: Fine by me, I was just making sure links were real links, and modules were real modules in the documentation ;-)

**Review by nitzmahone — APPROVED** [review]

**Comment by dharmabumstead** [comment]
Thanks @dagwieers and @nitzmahone!

**Pull request merged**